doc/nrf/libraries/dfu/dfu_target.rst

```diff
@@ -103,6 +104,67 @@ This DFU target downloads the serialized modem firmware to an external flash mem
 Once the modem firmware has been downloaded, the application should use :ref:`lib_fmfu_fdev` to write the firmware to the modem.
 The DFU target library does not perform the upgrade and calling the :c:func:`dfu_target_schedule_update` function has no effect.
 
+.. _lib_dfu_target_suit_style_update:
+
+SUIT-style upgrades
+-------------------
+
+SUIT-style firmware upgrades can be used for updates to the SUIT envelope and cache images.
+Depending on the image number, this type of DFU writes the data provided to the :c:func:`dfu_target_write` function into the following partitions:
+
+* Image 0: `dfu_partition`
+* Image 1: Reserved by SUIT to store the firmware included in the SUIT envelope.
```

Comment:
Is this true?
Images inside envelope are inside envelope, the dfu_cache_0 partition starts after the envelope and simple allows to reuse the unused part of the DFU partition.
I think it would be easier to point here to the dfu_cache_0 definition than clam that it is reserved.